src/_includes/layouts/form-vacature.liquid

```diff
@@ -21,15 +21,15 @@
                     <legend>Ik wil een vacature plaatsen</legend>
                     <p class="form-element">
                     <label for="form-1-input-0" class="form-label">Contactpersoon</label>
-                    <input type="text" name="Contactpersoon" id="form-1-input-0" class="text" required>
+                    <input type="text" name="Contactpersoon" id="form-1-input-0" class="text" autocomplete="name" required>
                     </p>
                     <p class="form-element">
                     <label for="form-1-input-1" class="form-label">E-mailadres contactpersoon</label>
-                    <input type="email" name="E-mailadres_contactpersoon" id="form-1-input-1" class="text" required>
+                    <input type="email" name="E-mailadres_contactpersoon" id="form-1-input-1" class="text" autocomplete="email work" required>
                     </p>
                     <p class="form-element">
                     <label for="form-1-input-2" class="form-label">Telefoonnummer</label>
-                    <input type="text" name="Telefoonnummer" id="form-1-input-2" class="text" required>
+                    <input type="text" name="Telefoonnummer" id="form-1-input-2" class="text" autocomplete="tel work" required>
```

Comment:
I think this is [order-dependent](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill) and may need to be `work tel`